### PR TITLE
PLT-3087 Added new segment analytics

### PR DESCRIPTION
**Pull request by DavidLu1997**
Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).

#### Summary
Added new segment analytics for
- User updated
- Nickname updated
- Username updated
- Email updated
- Theme updated
- Language updated

#### Unit Tests
N/A

#### Issue/Ticket Link
https://mattermost.atlassian.net/browse/PLT-3087

#### Has Critical Changes
- User updates

DO NOT MERGE until corresponding driver PR is in

### CODE REVIEW

File: webapp/components/user_settings/import_theme_modal.jsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ class ImportThemeModal extends React.Component {
         const user = UserStore.getCurrentUser();
         user.theme_props = theme;
 
-        Client.updateUser(user,
+        Client.updateUser(user, 'theme',
```
Comment (jwilander) [inline webapp/components/user_settings/import_theme_modal.jsx]: Minor but we should probably use constants for those strings, 'theme, 'language', etc.

**Comment by jwilander** [comment]
+1 other than minor comment

**Comment by crspeller** [comment]
Corresponding driver update: https://github.com/mattermost/mattermost-driver-javascript/pull/15

**Comment by crspeller** [comment]
Driver changes should be submitted and merged before PRs to platform are submitted. 
You need to have the hash to add to package.json.

**Pull request closed without merging**